proto/ssr.proto

```diff
@@ -54,9 +54,8 @@ message STECHeader {
     gnss.GpsTimeSec time = 1;
     uint32 num_msgs = 2;
     uint32 seq_num = 3;
-    uint32 ssr_update_interval = 4;
-    uint32 update_interval = 5;
-    uint32 iod_ssr = 6;
+    uint32 update_interval = 4;
+    uint32 iod_ssr = 5;
```

Comment:
yow: 
```
Non-required fields can be removed, as long as the field number is not used again in your updated message type. You may want to rename the field instead, perhaps adding the prefix "OBSOLETE_", or make the field number reserved, so that future users of your .proto can't accidentally reuse the number.
```
So if we rev a message that people are already using we can't delete a field unless we don't care about backwards compatibility?